discovery-provider/src/api/v1/tracks.py

```diff
@@ -129,40 +132,85 @@ class Track(Resource):
     @record_metrics
     @ns.doc(
         id="""Get Track""",
+        description="""Gets a track by its encoded ID""",
```

Comment:
Wanted to note, for external facing docs, I don't think we need to specify `encoded` because there is no notion of a non-encoded id 
In terms of understanding this from our team's perspective, the term should be here.